packages/thirdweb/src/extensions/modular/__generated__/BatchMetadataERC1155/write/uploadMetadata.ts

```diff
@@ -153,5 +153,6 @@ export function uploadMetadata(
       (await asyncOptions()).overrides?.maxPriorityFeePerGas,
     nonce: async () => (await asyncOptions()).overrides?.nonce,
     extraGas: async () => (await asyncOptions()).overrides?.extraGas,
+    erc20Value: async () => (await asyncOptions()).overrides?.erc20Value,
```

Comment:
What are these for?

Reply:
It's the new change from Joaquim to the generate script iirc - to add erc20Value to the transaction object.

I ran pnpm build:generate and these files were included - my guess is someone who added the modular extensions was running the generate script before the change from Joaquim